### ISSUE TITLE
Implement Tests

### ISSUE THREAD

**Issue by unnamedd** [unnamedd/swift-evolution-backend#17]

**Comment by tfrank64** [comment]
@unnamedd What does this task mean exactly? Just add more `XCTest` tests?

**Comment by unnamedd** [comment]
@tfrank64 yes. Today, we do not have any tests, will be awesome start to write tests for new features.

**Comment by tfrank64** [comment]
Ok, we have 4 tests already. We can definitely add more though.
